service/topology_coordinator.cc

```diff
@@ -842,6 +842,11 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
             rtlogger.trace("do update {} reason {}", updates, reason);
             mixed_change change{std::move(updates)};
             group0_command g0_cmd = _group0.client().prepare_command(std::move(change), guard, reason);
+            co_await utils::get_local_injector().inject("wait-before-committing-rf-change-event", [] (auto& handler) -> future<> {
+                rtlogger.info("wait-before-committing-rf-change-event injection hit");
```

Comment:
this should be debug log level or better not logged at all, error injector has its own logger: `logging::logger errinj_logger("debug_error_injection");`

Reply:
It's `info` lvl across the code base, from what I can see, so that tests, which use this log, don't have to switch to the debug/trace logging levels.

Reply:
it's clearly not info as those logs are purely used for debugging

Reply:
it matters for the readers and whoever will be debugging - debug level is for a reason

Reply:
That's the convention when using error injection. I'll assume it's a discussion on a separate problem, which can be continued elsewhere.